### ISSUE TITLE
Shields more button colour icons are different

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#18559]
## Description 
Shields more button colour icons are different

## Steps to reproduce 

   1. Install CR 95 build
   2. Open shields
   3. More buttons are of different colour

## Actual result 
![image](https://user-images.githubusercontent.com/17010094/136022533-30f67e2f-f53f-4cdb-91d0-5dd13caa9336.png)

## Expected result
![image](https://user-images.githubusercontent.com/17010094/136022711-1c2e5525-1237-41a4-91f5-3c5d9d8dc868.png)

## Issue reproduces how often 
Easy

## Version/Channel Information:

- Can you reproduce this issue with the current Play Store version? NA
- Can you reproduce this issue with the current Play Store Beta version? NA
- Can you reproduce this issue with the current Play Store Nightly version? Will be once CR95 is merged into master

## Device details
   - Install type (ARM, x86): ARM
   - Device type (Phone, Tablet, Phablet): Phone
   - Android version:11
   
## Brave version
1.31.36 CR 95 build

### Website problems only
- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards?NA
- Is the issue reproducible on the latest version of Chrome? NA

### Additional information

cc: @samartnik

**Comment by samartnik** [comment]
Fixed directly in `cr95` branch https://github.com/brave/brave-core/commit/1313149ea969ee56bc9a5ecd0099218ae6aedcc2

**Comment by stephendonner** [comment]
Verified `PASSED` using Brave Beta `1.32.94, Chromium 95.0.4638.69` on a Google Pixel XL, Android 9.

example | example
---------|---------
![Screenshot_20211108-093925](https://user-images.githubusercontent.com/387249/140791555-22daf7a7-4855-4cda-a077-e94a7660fc1c.png) | ![Screenshot_20211108-094028](https://user-images.githubusercontent.com/387249/140791562-7e61c59a-1e29-45b4-81f8-8fbfb8968ed5.png)
